### PR TITLE
Fix with OpenCascade 7.8.0

### PR DESCRIPTION
**Pull request by thierry-FreeBSD**
This resolves https://github.com/f3d-app/f3d/issues/1304 .

### CODE REVIEW

**Comment by thierry-FreeBSD** [comment]
It failed! It seems that more patches must be added to support OCCT previous release!

File: plugins/occt/CMakeLists.txt [hunk 1]
```diff
@@ -27,8 +33,8 @@ option(F3D_PLUGIN_OCCT_COLORING_SUPPORT "Enable coloring support in occt plugin"
 mark_as_advanced(F3D_PLUGIN_OCCT_COLORING_SUPPORT)
 
 if(F3D_PLUGIN_OCCT_COLORING_SUPPORT)
-  if (NOT (TARGET "TKXDESTEP") OR NOT (TARGET "TKXDEIGES"))
-      message(FATAL_ERROR "occt plugin: TKXDESTEP and TKXDEIGES OCCT modules are not found. Turn off F3D_PLUGIN_OCCT_COLORING_SUPPORT or enable them in your OpenCascade build.")
+  if (NOT (TARGET "TKXSDRAWSTEP") OR NOT (TARGET "TKXSDRAWIGES"))
```
Comment (mwestphal) [inline plugins/occt/CMakeLists.txt:36]: the name of the modules changes, you need to check the version first before checking. I'd suggest putting the names in a CMake list.

**Comment by mwestphal** [comment]
Hi @thierry-FreeBSD ,

Any reason why you closed this ? It justs needs a litle bit more work :)

**Comment by thierry-FreeBSD** [comment]
I know how to submit a PR, but I don't know how to modify it!

**Comment by mwestphal** [comment]
Its actually pretty easy, just commit and push more commits :)

**Pull request closed without merging**